compiler/util/files.cpp

```diff
@@ -440,6 +440,18 @@ std::string runPrintChplEnv(std::map<std::string, const char*> varMap) {
   return runCommand(command);
 }
 
+std::string getChplPythonVersion() {
+  // Runs util/chplenv/chpl_python_version.py and left-trims the string
+
+  std::string command = "";
+  command += std::string(CHPL_HOME) + "/util/chplenv/chpl_python_version.py 2> /dev/null";
+
+  std::string pyVer = runCommand(command);
+  pyVer.erase( pyVer.find_last_not_of("\n\r")+1);
```

Comment:
Remove extra space before `pyVer.` if you're going to be editing this file anyways